.github/workflows/integration-tests.yaml

```diff
@@ -82,52 +82,13 @@ jobs:
           echo "KEPTN_API_TOKEN=${{ steps.install_keptn.outputs.KEPTN_API_TOKEN }}" >> $GITHUB_ENV
           go mod download
 
-      #########################################################################
-      #                           Integration Tests                           #
-      #########################################################################
-      - name: Run "Hello World" test
-        id: test_hello_world
-        continue-on-error: true
-        working-directory: test/e2e
-        run: go test -v -run "^\QTestHelloWorldDeployment\E$"
-
-      - name: Run "Files" test
-        id: test_files
-        continue-on-error: true
-        working-directory: test/e2e
-        run: go test -v -run "^\QTestResourceFiles\E$"
-
-      - name: Run "Env. variables" test
-        id: test_env_vars
-        continue-on-error: true
-        working-directory: test/e2e
-        run: go test -v -run "^\QTestEnvironmentVariables\E$"
-
-      - name: Run "Job cleanup" test
-        if: false
-        id: test_job_cleanup
-        continue-on-error: true
-        working-directory: test/e2e
-        run: go test -v -run "^TestJobCleanup*"
-
-      - name: Run "Locust" test
-        id: test_locust
-        continue-on-error: true
-        working-directory: test/e2e
-        run: go test -v -run "^\QTestLocust\E$"
-
-      #########################################################################
-
-      # Collect all test reports of the matrix in report files
-      - name: Write test report
-        if: always()
+      - name: Run Integration tests
         env:
           TEST_REPORT_FILENAME: test-report-${{ github.run_id }}-${{ matrix.keptn-version }}.json
         run: |
-          echo "Write test report to $TEST_REPORT_FILENAME"
-          TEST_RESULT_JSON=$(echo '${{ toJson(steps) }}' | jq '. | with_entries( select(.key|contains("test_"))) | to_entries | map( {name: .key, outcome: .value.outcome} )')
-          
-          echo $TEST_RESULT_JSON > $TEST_REPORT_FILENAME
+          TEST_OUTPUT=$(go test ./test/e2e/... -json | grep -P "skip|pass|fail" | grep '"Test":')
```

Comment:
I would feel more comfortable if we still parse the output using `jq` :sweat_smile: